spark-bigquery-connector-common/src/test/java/com/google/cloud/spark/bigquery/integration/WriteIntegrationTestBase.java

```diff
@@ -93,18 +96,36 @@ abstract class WriteIntegrationTestBase extends SparkBigQueryIntegrationTestBase
   protected final SparkBigQueryConfig.WriteMethod writeMethod;
   protected Class<? extends Exception> expectedExceptionOnExistingTable;
   protected BigQuery bq;
+  protected boolean sparkVersionAboveOrEqualTo3_4;
+
+  protected DataType timeStampNTZType;
 
   public WriteIntegrationTestBase(SparkBigQueryConfig.WriteMethod writeMethod) {
-    this(writeMethod, IllegalArgumentException.class);
+    this(writeMethod, IllegalArgumentException.class, false, null);
+  }
+
+  public WriteIntegrationTestBase(
```

Comment:
No need for the flag, see Spark34ReadIntegrationTest for suggestion

Reply:
Fixed